scripts/styleCheck.fsx

```diff
@@ -0,0 +1,192 @@
+            },
+            Echo.Off
+        )
+        .UnwrapDefault()
+    |> ignore
+
+    let processResult =
+        Process.Execute(
+            {
+                Command = "git"
+                Arguments = "diff --exit-code"
+            },
+            Echo.Off
+        )
+
+    processResult
+
+let GitRestore() =
+    Process
+        .Execute(
+            {
+                Command = "git"
+                Arguments = "restore ."
+            },
+            Echo.Off
+        )
+        .UnwrapDefault()
+    |> ignore
+
+let CheckStyleOfFSharpFiles(rootDir: DirectoryInfo) : bool =
+    let suggestion =
+        Some "Please style your F# code using: `dotnet fantomless --recurse .`"
+
+    GitRestore()
+
+    let success =
+        if ContainsFiles rootDir "*.fs" || ContainsFiles rootDir ".fsx" then
+            StyleFSharpFiles rootDir
+            let processResult = GitDiff()
+            UnwrapProcessResult suggestion true processResult |> ignore
+            IsProcessSuccessful processResult
+
+        else
+            true
+
+    success
+
+let CheckStyleOfTypeScriptFiles(rootDir: DirectoryInfo) : bool =
+    let pattern =
+        $".{Path.DirectorySeparatorChar}**{Path.DirectorySeparatorChar}*.ts"
+
+    let suggestion =
+        Some
+            $"Please style your TypeScript code using: `npx prettier --quote-props=consistent --write {pattern}`"
+
+    GitRestore()
+
+    let success =
+        if ContainsFiles rootDir "*.ts" then
+            StyleTypeScriptFiles()
+            let processResult = GitDiff()
+            UnwrapProcessResult suggestion true processResult |> ignore
+            IsProcessSuccessful processResult
+
+        else
+            true
+
+    success
+
+let CheckStyleOfYmlFiles(rootDir: DirectoryInfo) : bool =
+    let pattern =
+        $".{Path.DirectorySeparatorChar}**{Path.DirectorySeparatorChar}*.yml"
+
+    let suggestion =
+        Some
+            $"Please style your YML code using: `npx prettier --quote-props=consistent --write {pattern}`"
+
+    GitRestore()
+
+    let success =
+        if ContainsFiles rootDir "*.yml" then
+            StyleYmlFiles()
+            let processResult = GitDiff()
+            UnwrapProcessResult suggestion true processResult |> ignore
+            IsProcessSuccessful processResult
+        else
+            true
+
+    success
+
+let CheckStyleOfCSharpFiles(rootDir: DirectoryInfo) : bool =
+    let suggestion =
+        Some
+            "Please style your C# code using: `dotnet format whitespace . --folder"
+
+    GitRestore()
+
+    let success =
+        if ContainsFiles rootDir "*.cs" then
+            StyleCSharpFiles rootDir
+            let processResult = GitDiff()
+            UnwrapProcessResult suggestion true processResult |> ignore
+            IsProcessSuccessful processResult
+        else
+            true
+
+    success
+
+let CheckStyleOfXamlFiles(rootDir: DirectoryInfo) : bool =
+    let prettierPath =
```

Comment:
@realmarv this path is not being used by the script, just being shown as an error message to the user; therefore it should not use Directory.GetCurrentDirectory(); it is a relative path that should STAY relative because you want to just show it to the user